toolchain/lex/tokenized_buffer.h

```diff
@@ -179,6 +162,13 @@ class TokenizedBuffer : public Printable<TokenizedBuffer> {
   // Returns the previous line handle.
   auto GetPrevLine(LineIndex line) const -> LineIndex;
 
+  // Returns true if the token comes after the comment.
+  auto IsBeforeComment(TokenIndex token, CommentIndex comment_index) const
+      -> bool;
```

Comment:
I read `buffer.IsBeforeComment(token, comment_index)` as querying whether "`token` is before `comment_index`". Could we make this implement that API? Or be named `IsAfterComment` so that it reads "`token` is after `comment`"?

Reply:
Fixed to be `IsAfterComment`, that was an error in naming.

Note, I've also removed the base PRs so that this can be merged.